src/Articles/ArticleCard.jsx

```diff
@@ -0,0 +1,22 @@
+const ArticleCard = ({articles}) => {
+    console.log(articles)
+    return (
+        <ol>
+                {articles.map((article) => {
```

Comment:
It might be best to put your "ol" and map in your ArticlesList component and then just have the ArticleCard be the "li" element (and everything inside it)